bowtie/_connectables.py

```diff
@@ -76,6 +78,7 @@ class Connectable:
         (cls.connector, cls)
         for cls in [
             _containers.ConnectableImage,
+            _containers.ConnectedContainer,
```

Comment:
Can you make the name of this match the other one? I.e. `ConnectableContainer` not `Connected` (the connection hasn't happened yet by the time this object is created).